### PR TITLE
fix(PPDSC-2254): update audio file used in tests

### PR DESCRIPTION
**Pull request by tbradbury**
PPDSC-2254

**What**

1. Background - why this is needed - Tests are failing because link to audio being used has stopped working
2. What did you do - added audio files (used some Wireless audio thought it best to be owned by the company) to the static folder so we can use them in tests. Update tests to work with new audio
3. What does the reviewers should expect - Tests to pass

**I have done:**
- [x] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [x] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2254-audio
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2254-audio/storybook

**Comment by evgenitsn** [comment]
In `src/audio-player/__tests__/auto-play-audio-player.stories.tsx` there is a ` src="/audio-player-song.mp3"` which is the original file that we had (the short one). 
Can we delete that file and use some of the new audio files there as well?

**Review by evgenitsn — APPROVED** [review]

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <meta name="viewport" content="width=device-width, initial-scale=1" />
+<base target='_top' />
```
Comment (mutebg) [inline .storybook/preview-head.html:2]: Just curious what is this doing and why we need it?

**Review by mutebg — APPROVED** [review]

File: .storybook/preview-head.html (hunk 1 above)
Comment (tbradbury) [reply .storybook/preview-head.html:2]: It was an attempt to fix audio on PR environment. It did not work and I'll be reverting it.

**Review by mutebg — APPROVED** [review]

**Pull request merged**